suite-native/app/app.config.ts

```diff
@@ -141,6 +141,7 @@ export default ({ config }: ConfigContext): ExpoConfig => {
     return {
         ...config,
         name,
+        scheme: 'trezorsuitelite',
```

Comment:
since there is a plan to remove Lite from name once app reaches full custody, wouldnt it be better to register `trezorsuite` scheme already instead?

Reply:
It's not planned used custom deep link scheme at all, we will use Universal Links for prod variant but it's much more complicated to setup so we will use this for development